### PR TITLE
SW-2338 Withdrawal Log Details Photos

### PR DESCRIPTION
**Pull request by charlie-tf**
- do not show Reassign button if there are no plots
- query for photo ids
- display the photos in a thumbnail
- show the photos in a carousel when clicked

image
image

### CODE REVIEW

File: src/components/NurseryWithdrawals/NurseryWithdrawalsDetails.tsx [hunk 1]
```diff
@@ -163,7 +165,7 @@ export default function NurseryWithdrawalsDetails({
             <Typography color={theme.palette.TwClrTxt} fontSize='20px' fontWeight={600} fontStyle='italic'>
               {withdrawal?.withdrawnDate}
             </Typography>
-            {withdrawal?.purpose === 'Out Plant' && (
+            {withdrawal?.purpose === 'Out Plant' && hasPlots && (
```
Comment (karthikbtf) [inline src/components/NurseryWithdrawals/NurseryWithdrawalsDetails.tsx:168]: thankyou!

File: src/components/common/ViewPhotosModal.tsx [hunk 2]
```diff
@@ -95,10 +94,10 @@ export default function ViewPhotosDialog(props: ViewPhotosDialogProps): JSX.Elem
           ssr={true}
           afterChange={disableButtons}
         >
-          {photosNames.map((photoName, i) => (
+          {photosUrls.map((url, i) => (
```
Comment (karthikbtf) [inline src/components/common/ViewPhotosModal.tsx:97]: Awesome, thanks for the refactor!

**Review by karthikbtf — COMMENTED** [review]
Looks awesome.
The refactor is great for view photos modal.
Just a sanity check - confirm this works for the accessions view?

**Comment by charlie-tf** [comment]
>Just a sanity check - confirm this works for the accessions view?

@karthikbtf Yup, I checked it out! Thanks!

**Comment by karthikbtf** [comment]
> > Just a sanity check - confirm this works for the accessions view?
> 
> @karthikbtf Yup, I checked it out! Thanks!

Thanks!!

**Review by karthikbtf — APPROVED** [review]

File: src/components/NurseryWithdrawals/WithdrawalDetails/sections/Photos.tsx [hunk 3]
```diff
@@ -1,15 +1,79 @@
...
+        onClose={() => setPhotosModalOpened(false)}
+        selectedSlide={selectedSlide}
+      />
       <Typography fontSize='20px' fontWeight={600}>
         {strings.PHOTOS}
       </Typography>
-      <Box>(Photos go here)</Box>
+      <Box display='flex' flexWrap='wrap'>
+        {photoUrls.map((photoUrl, index) => (
+          <Box
```
Comment (karthikbtf) [inline src/components/NurseryWithdrawals/WithdrawalDetails/sections/Photos.tsx:59]: @charlie-tf this looks great in staging. Can you add a 'cursor: pointer' style to this?

Comment (charlie-tf) [reply src/components/NurseryWithdrawals/WithdrawalDetails/sections/Photos.tsx:59]: can do!

**Pull request merged**